### ISSUE TITLE
Add "BodyText" style to docx?

### ISSUE THREAD

**Issue by jkr** [jgm/pandoc#1937]
Right now, the docx stylesheet has "Normal" as its base paragraph style, and also the style of its body text. This means, for example, that if you add a first-line indent to body text, it also gets added to abstract, headers, footnotes, and everything else that inherits from "Normal."

How about having body text use the "BodyText" style, and have a hierarchy more like:

~~~
Normal
  |---- Title
         Heading N
         Footnote Text
         BodyText
           |----- FirstParagraph
                   BlockQuote
                   etc
~~~~

So you could change the document font with normal, but changing BodyText wouldn't mess with section headings and so on.

**Comment by jkr** [comment]
I should have added that I'd be happy to implement this.

**Comment by jgm** [comment]
Sure, I'd have no objections.

+++ Jesse Rosenthal [Feb 10 15 17:32 ]:

**Comment by jkr** [comment]
Closed with f703198bc94c51fed15f7e3e9784bac88cc0818f
